scss/_bs4alphacompat.scss

```diff
@@ -26,23 +26,23 @@ $danger: $red !default;
 $light: $gray-100 !default;
 $dark: $gray-800 !default;
 
-$brand-success: $success;
-$brand-info: $info;
-$brand-warning: $warning;
-$brand-danger: $danger;
+$brand-success: $success !default;
```

Comment:
All of these should be !default so that the Snap theme can override them with it's own values. You'll notice that before this fix the Alert notifications for success, info, warning and danger don't have the original Snap colours pre BS4 (Moodle 3.5)